db/migrate/20181127165847_add_show_replies_to_lists.rb

```diff
@@ -0,0 +1,23 @@
+require Rails.root.join('lib', 'mastodon', 'migration_helpers')
+
+class AddShowRepliesToLists < ActiveRecord::Migration[5.2]
+  include Mastodon::MigrationHelpers
+
+  disable_ddl_transaction!
+
+  def up
+    safety_assured do
+      add_column_with_default(
+        :lists,
+        :replies_policy,
```

Comment:
This is the first enum in our code we call `_policy`, I don't want to bikeshed too much but is this a fully appropriate name?

Reply:
This has been in glitch-soc for almost two years, so I'd need to check if any client uses that API, but otherwise I'm fine with changing the name if you have a better suggestion.